QKSMS/src/main/res/values/strings.xml

```diff
@@ -333,8 +333,12 @@
     <string name="restart_message">These changes will take effect the next time QKSMS is opened</string>
     <!--Dialog title for asking if a user wants to delete a conversation-->
     <string name="delete_conversation">Delete conversation</string>
+    <!--Dialog title for asking if a user wants to delete all failed messages-->
+    <string name="delete_all_failed">Delete conversation</string>
```

Comment:
The string says "Delete conversation" but it should specify that it only deletes failed messages